extensions/interactions/base_test.py

```diff
@@ -254,6 +254,7 @@ def test_default_interactions_are_valid(self):
             # The interaction directory should contain the following files:
             #  Required:
             #    * A python file called {InteractionName}.py.
+            #    * An __init__.py file used to import the Python file.
```

Comment:
Shouldn't there be a corresponding isfile() assert somewhere for this? (Read the entirety of this test.)

Reply:
Done